### ISSUE TITLE
TaskCanceledException when executing performance tests with 250 users

### ISSUE THREAD

**Issue by abdelbena** [kblok/puppeteer-sharp#417]
### Description

I'm trying to run performance tests with 250 users using visual studio load tests on azure. 

The code I'm trying to test:
```
                var browser = await Puppeteer.LaunchAsync(new LaunchOptions { UserDataDir = 
                                     Settings.UserDataDir, ExecutablePath = Settings.ChromiumExecutablePath, 
                                     Headless = true, Timeout = 0, KeepAliveInterval = 600 });

                using (var page = await browser.NewPageAsync())
                {

                    page.DefaultNavigationTimeout = Timeout; //set navigation timeout
                    await page.GoToAsync(url, 0);
                    result = await page.PdfStreamAsync(new PdfOptions
                    {
                        PrintBackground = true
                    });
                }

                await browser.CloseAsync(); 
```

### Expected behavior:
Return pdf and close chromium process

### Actual behavior:
More than 80 requests failed with the exception System.Threading.Tasks.TaskCanceledException at PuppeteerSharp.Helpers.TaskQueue+<Enqueue>d__3.MoveNext

Message : The 'System.Net.WebSockets.InternalClientWebSocket' instance cannot be used for communication because it has been transitioned into the 'Aborted' state.
### Versions

* Puppeteer 1.2.0
* .NET Framework 4.7.2.

### Additional Information

Other exception are thrown : System.Net.WebSockets.WebSocketException at PuppeteerSharp.Helpers.TaskQueue+<Enqueue>d__3.MoveNext

Message: The 'System.Net.WebSockets.InternalClientWebSocket' instance cannot be used for communication because it has been transitioned into the 'Aborted' state.

**Comment by viswa18** [comment]
@kblok  I'm also getting this exception in v1.5 can you please let me know is there any solution for this.

**Comment by abdelbena** [comment]
@kblok when I embedded puppeteer-sharp source code in my solution it works fine without exception, but when I use the nuget package I got the exception.

**Comment by kblok** [comment]
@abdelbena which Nuget package version?

**Comment by abdelbena** [comment]
@kblok I was using 1.2.0

**Comment by kblok** [comment]
@abdelbena there were many task management improvements on v1.5 that will help.

**Comment by kblok** [comment]
Closed due to inactivity. Feel free to reopen it if needed.

**Comment by abdelbena** [comment]
@kblok yes the issue was fixed from 1.2.0. sorry to not be able to reply before.
